src/pages/Account/TokenIcon.tsx

```diff
@@ -40,8 +40,13 @@ export function TokenIcon({ symbol, address, chainId, width = 32, height = 32, m
   }
 
   const urlSource = getNetworkDefaultTokenUrl(symbol, token?.logoURI)
+
   if (urlSource) {
-    sources.push(urlSource)
+    if (token?.symbol === WETH[ChainId.MAINNET].symbol) {
+      sources.push(getTokenLogoURL(WETH[ChainId.MAINNET].address, ChainId.MAINNET))
+    } else {
+      sources.push(urlSource)
+    }
```

Comment:
I just wanted use a different icon for `WETH`.
Cannot use `address` because `const token = allTokens.get(symbol)` and it returns first `WETH` from allTokens lists

Reply:
Understood :)